dev-docs/bidders/connatix.md

```diff
@@ -24,30 +24,111 @@ ortb_blocking_supported: true
 sidebarType: 1
 ---
 
+<div style="height: 2px; background-color: #333333; width: 100%" ></div>

```

Comment:
```suggestion
```

Those custom HTML is not needed. This makes it less consistent with the rest of the docs and thus less consumable.

Reply:
Deleted the div separators.